examples/runners/docker/basic.rst

```diff
@@ -81,6 +81,8 @@ Now, we need to define two new profiles inside the conan ``profiles`` folder. Re
     dockerfile=</my/runner/folder>/mylib
     cache=copy
     remove=true
+    # Add the following line if running on a Mac Silicon
+    platform=linux/amd64
```

Comment:
This sounds confusing, as the arch would also change to `arch=armv8`. I think it'd be better to say this within a note below or something similar